packages/kcms/src/match/model/main.ts

```diff
@@ -13,10 +14,21 @@ export interface CreateMainMatchArgs {
   teamId2?: TeamID;
   winnerId?: TeamID;
   runResults: RunResult[];
+  parentMatchID: MainMatchID | undefined;
+  childMatches: ChildrenMatches | undefined;
+}
+
+/**
+ * 自分より前に行われた試合
+ */
+export interface ChildrenMatches {
+  match1: MainMatch;
+  match2: MainMatch;
 }
 
 /*
  * @description 本戦の試合
+ * parentIDとchildrenMatchesがそれぞれundefinedになることはない
```

Comment:
```suggestion
 * parentIDとchildrenMatchesがともにundefinedになることはない
```
「それぞれ」は「個別に・独立して」の意味があります